images/win/Windows2019-Readme.md

```diff
@@ -1,107 +1,122 @@
+| Announcements |
+|-|
+| [[ubuntu, windows] Default Go version will be set to 1.15 on February, 8](https://github.com/actions/virtual-environments/issues/2447) |
+| [[all OSs] Java 7 will be removed from images on February, 8.](https://github.com/actions/virtual-environments/issues/2446) |
+| [[all OS versions] Android NDK is updated to r22.](https://github.com/actions/virtual-environments/issues/2420) |
+| [[Windows] Default PHP is set to 8.0](https://github.com/actions/virtual-environments/issues/2416) |
+***
 # Microsoft Windows Server 2019 Datacenter
-- OS Version: 10.0.17763 Build 1637
-- Image Version: 20210110.1
+- OS Version: 10.0.17763 Build 1697
+- Image Version: 20210118.1
 
 ## Enabled windows optional features
 - Windows Subsystem for Linux [WSLv1]
 
 ## Installed Software
 ### Language and Runtime
-- Java 1.7.0_232 
-- Java 1.8.0_275 (default)
-- Java 11.0.9.1 
-- Java 13.0.2 
 - Bash 4.4.23(1)-release
-- Python 3.7.9
-- Ruby 2.5.8p224
 - Go 1.14.13
-- PHP 8.0.1
+- Java 1.7.0_232  Java 1.8.0_275 (default) Java 11.0.9.1  Java 13.0.2 
```

Comment:
looks like all java versions were joined to the single line but it is not big deal:
It was fixed in https://github.com/actions/virtual-environments/pull/2467